producer.robot

```diff
@@ -11,7 +11,7 @@ Produce items
     ...    Get source Excel file from work item.
     ...    Read rows from Excel.
     ...    Creates output work items per row.
-    ${path}=    Get Work Item File    orders.xlsx    %{ROBOT_ARTIFACTS}${/}orders.xlsx
+    ${path}=    Get Work Item File    orders.xlsx    %{ROBOT_ARTIFACT=temp}${/}orders.xlsx
```

Comment:
I want to learn more: What's with the `ROBOT_ARTIFACT` env var here, as is the first time I'm seeing this? :D

Is it something injected by CR which points to a place from where the artifact files will be uploaded after the Step run finishes? If yes, then wouldn't be `${OUTPUT_DIR}` a better option here? (as an opportunistic refactoring)

Reply:
I honestly had no idea where this env variable came from, but figured to avoid problems, I'd give it a default. The `env` files didn't have it before I deleted them and it's not a built in or automatic variable.

Reply:
Okay, I booted up the interactive console and was able to confirm that `%{ROBOT_ARTIFACTS}` is a built in environment variable (from our tooling maybe?) that points to the `${OUTPUT DIR}`. So this change of mine is unnecessary.

Reply:
Agreed, and pushed